gulpfile.babel.js

```diff
@@ -275,11 +275,11 @@ gulp.task('default', ['clean'], cb => {
 // Build production files and microsite
 gulp.task('all', ['clean'], cb => {
   runSequence(
-    ['default', 'styletemplates'],
-    ['styles:gen'],
-    ['lint', 'scripts', 'assets', 'demos', 'pages',
-     'templates', 'images', 'styles-grid', 'metadata'],
+    ['styles:gen', 'styles-grid', 'styletemplates'],
```

Comment:
Alright, I'll just move that up into its own block then. Easy fix. I didn't see any side-effects, but it could also be a race-condition that I didn't hit. Better safe than sorry here. :wink: